sw/host/cryptotest/testvectors/data/schemas/BUILD

```diff
@@ -13,3 +13,8 @@ filegroup(
     name = "ecdsa_sig_ver_schema",
     srcs = ["ecdsa_sig_ver_schema.json"],
 )
+
+filegroup(
```

Comment:
Can we change this to use `exports_files` instead of `filegroup`? (See https://github.com/lowRISC/opentitan/pull/21193#discussion_r1478891197)